### PR TITLE
Convert AggregatedFeedContent related classes to Kotlin

### PR DESCRIPTION
**Pull request by cooltey**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/feed/FeedCoordinatorBase.java [hunk 1]
```diff
@@ -301,7 +300,6 @@ private boolean isDailyCardType(@NonNull Card card) {
 
     private boolean shouldShowProgressCard(@NonNull FeedClient pendingClient) {
         return pendingClient instanceof SuggestedEditsFeedClient
-                || pendingClient instanceof AggregatedFeedContentClient
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/feed/FeedCoordinatorBase.java:304]: @cooltey why was this removed?

Comment (cooltey) [reply app/src/main/java/org/wikipedia/feed/FeedCoordinatorBase.java:304]: Because `AggregatedFeedContentClient` is not extended from `FeedClient`, the logic is unnecessary here at the beginning.
